app/code/ZP/LoyaltyProgram/Model/Customer/Email/Manager.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+namespace ZP\LoyaltyProgram\Model\Customer\Email;
+
+use Magento\Customer\Api\Data\CustomerInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\MailException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\StoreManagerInterface;
+use ZP\LoyaltyProgram\Model\Configs\Program\Scope\Email\Config;
+use ZP\LoyaltyProgram\Model\LoyaltyProgram;
+use Magento\Framework\App\Area;
+
+class Manager
+{
+    private const SENDER_EMAIL = 'mailhog@gmail.com';
+    private const SENDER_NAME = 'MailHog';
+
+    public function __construct(
+        private Config $emailConfig,
+        private TransportBuilder $transportBuilder,
+        private StoreManagerInterface $storeManager,
+    ) {}
+
+    /**
+     * @param CustomerInterface $customer
+     * @param LoyaltyProgram $loyaltyProgram
+     * @throws LocalizedException
+     * @throws MailException
+     * @throws NoSuchEntityException
+     * @throws \Exception
+     */
+    public function sendProgramAssignmentEmailMsg(CustomerInterface $customer, LoyaltyProgram $loyaltyProgram): void
+    {
+        if ($this->isEnabledToSendEmailToCustomer((int)$customer->getWebsiteId())) {
+            $this->sendEmailMessage($customer, $loyaltyProgram);
+        }
+    }
+
+    /**
+     * @param int $websiteId
+     * @return bool
+     * @throws \Exception
+     */
+    private function isEnabledToSendEmailToCustomer(int $websiteId): bool
+    {
+        return $this->emailConfig->isEnabled(Config::GENERAL_GROUP_CONFIG_TYPE, $websiteId) &&
```

Comment:
what is a purpose of two is enabled calls?

Reply:
The general group field is_enable response for is email sending allowed or not in context of Loyalty Programs.
The customer group field is_enable response for is it allowed to send emails to customers about Loyalty Programs.
I understand that may be it is not necessary now to have a 'general is_enable config' and 'customer is_enable config', but I decided that this few code lines will be a nice preparation for future extension of mail sending for other entities( like  admin , companies etc.) in context about Loyalty Programs. If you want, I can remove this, and leave only general group is_enabled config, or customer group is_enabled config.

Reply:
please leave only general group in admin system configuration